packages/wonder-stuff-ci/src/extract-mobile-release-info-from-branch-name.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Extract release version and prefix from mobile release branch.
+ *
+ * Example, given the branch `release/unified/v7.8.0`:
+ * {
+ *    prefix: "release/unified/",
+ *    version: "7.8.0"
+ * }
+ *
+ * @param {string} The release branch of the form `[release/[unified|ios|android]]/[v]<num>.<num>.<num>[-extra]`.
+ * @returns {Object} The release version and prefix, if found; otherwise, `null`.
+ */
+export const extractMobileReleaseInfoFromBranchName = (arg: string | null) => {
```

Comment:
**suggestion:** Currently, the return type is inferred from the code and as such, there isn't a concrete way to refer to this type. It would be helpful for calling code if there was one as it can be used by them to type their own arguments and variables.

Something like:

```
type MobileReleaseInfo = {
    prefix: "release/unified/" | "release/ios" | "release/android",
    version: string,
};
```

And then:
```suggestion
export const extractMobileReleaseInfoFromBranchName =
    (arg: string | null): MobileReleaseInfo => {
```